### ISSUE TITLE
arm zip file seems incorrect

### ISSUE THREAD

**Issue by nsteinmetz** [influxdata/telegraf#562]
Hi,

On my RPI2B, what I have when I try to decompress the arm file for telegraf.

File is correct:
```
pi@cian ~ $ md5sum telegraf-0.10.0_linux_arm.tar.gz 
6aa63d6af75fdee954a0f204b4c9a8af  telegraf-0.10.0_linux_arm.tar.gz
```

But, when decompressing:

```
pi@cian ~ $ tar tzf telegraf-0.10.0_linux_arm.tar.gz 
gzip: stdin has more than one entry--rest ignored
tar: Child returned status 2
tar: Error is not recoverable: exiting now
```

Seems more a zip archive:

```
pi@cian ~ $ file telegraf-0.10.0_linux_arm.tar.gz 
telegraf-0.10.0_linux_arm.tar.gz: Zip archive data, at least v1.0 to extract
```

And so:

```
pi@cian ~ $ unzip telegraf-0.10.0_linux_arm.tar.gz 
Archive:  telegraf-0.10.0_linux_arm.tar.gz
   creating: tmp/package-dir-staging20160111-17268-1nv9whc/
   creating: tmp/package-dir-staging20160111-17268-1nv9whc/etc/
   creating: tmp/package-dir-staging20160111-17268-1nv9whc/etc/logrotate.d/
  inflating: tmp/package-dir-staging20160111-17268-1nv9whc/etc/logrotate.d/telegraf  
   creating: tmp/package-dir-staging20160111-17268-1nv9whc/etc/telegraf/
  inflating: tmp/package-dir-staging20160111-17268-1nv9whc/etc/telegraf/telegraf.conf  
   creating: tmp/package-dir-staging20160111-17268-1nv9whc/usr/
   creating: tmp/package-dir-staging20160111-17268-1nv9whc/usr/bin/
  inflating: tmp/package-dir-staging20160111-17268-1nv9whc/usr/bin/telegraf  
   creating: tmp/package-dir-staging20160111-17268-1nv9whc/usr/lib/
   creating: tmp/package-dir-staging20160111-17268-1nv9whc/usr/lib/telegraf/
   creating: tmp/package-dir-staging20160111-17268-1nv9whc/usr/lib/telegraf/scripts/
  inflating: tmp/package-dir-staging20160111-17268-1nv9whc/usr/lib/telegraf/scripts/init.sh  
  inflating: tmp/package-dir-staging20160111-17268-1nv9whc/usr/lib/telegraf/scripts/telegraf.service  
   creating: tmp/package-dir-staging20160111-17268-1nv9whc/var/
   creating: tmp/package-dir-staging20160111-17268-1nv9whc/var/log/
   creating: tmp/package-dir-staging20160111-17268-1nv9whc/var/log/telegraf/
```

Could you please review the way you create the archive?

Thanks,
Nicolas

**Comment by rossmcdonald** [comment]
@nsteinmetz Thanks, this was a bug in our packaging script. The `zip` was just renamed to `tar.gz` when it was exported. I should have a fix for this later today.

**Comment by sparrc** [comment]
@rossmcdonald do you have a fix for this?

**Comment by rossmcdonald** [comment]
@sparrc Yes, working on it now. Got distracted with some compilation issues due to the import path change.

**Comment by rossmcdonald** [comment]
Fixed by #578
